### PR TITLE
12.0 mig web translate dialog

### PR DESCRIPTION
**Pull request by grindtildeath**

### CODE REVIEW

File: web_translate_dialog/static/src/js/web_translate_dialog.js [hunk 1]
```diff
@@ -121,36 +117,36 @@ var translateDialog = Dialog.extend({
...
         this.$el.find('.oe_translation_field').val('').removeClass('touched');
-        _.each(self.languages, function(lg) {
-            var context = new Context(session.user_context, {lang: lg.code});
-            var deff = $.Deferred();
-            deferred.push(deff);
-            rpc.query({
-            model: self.view.modelName,
-            method: 'read',
+
+        //var context = new Context(session.user_context, {lang: lg.code});
```
Comment (tschanzt) [inline web_translate_dialog/static/src/js/web_translate_dialog.js]: :scissors:

File: web_translate_dialog/static/src/js/web_translate_dialog.js [hunk 2]
```diff
@@ -121,36 +117,36 @@ var translateDialog = Dialog.extend({
...
+            });
+
         return deferred;
     },
     on_button_save: function() {
         var translations = {},
             self = this,
             save_mutex = new Mutex();
-        self.$el.find('.oe_translation_field.touched').each(function() {
+        this.$el.find('.oe_translation_field.touched').each(function() {
```
Comment (tschanzt) [inline web_translate_dialog/static/src/js/web_translate_dialog.js:148]: uneeded change

**Review by tschanzt — APPROVED** [review]
only some minor nitpicks nothing major

File: web_translate_dialog/static/src/js/web_translate_dialog.js (hunk 2 above)
Comment (grindtildeath) [reply web_translate_dialog/static/src/js/web_translate_dialog.js:148]: Isn't it 'better' to use `this` if it's available ?

Comment (TDu) [reply web_translate_dialog/static/src/js/web_translate_dialog.js:148]: I would say so too, here the use of `self` is not needed AFAIK

**Review by TDu — APPROVED** [review]
LG, but I am a little out of context

**Review by simahawk — APPROVED** [review]

**Pull request merged**